recipe/meta.yaml

```
@@ -1,6 +1,6 @@
 {% set name = "cuda-cccl" %}
 {% set version = "12.4.99" %}
-{% set impl_version = "2.0.1" %}
+{% set impl_version = "2.3.1" %}
```

Maybe we should rename this:
```suggestion
{% set cccl_version = "2.3.1" %}
```